app/src/composables/use-item/use-item.ts

```diff
@@ -317,36 +317,7 @@ export function useItem(collection: Ref<string>, primaryKey: Ref<string | number
 			_and: [],
 		} as LogicalFilterAND;
 
-		const applyConditions = (field: Field) => {
-			if (field.meta && Array.isArray(field.meta?.conditions)) {
-				const conditions = [...field.meta.conditions].reverse();
-
-				const matchingCondition = conditions.find((condition) => {
-					if (!condition.rule || Object.keys(condition.rule).length !== 1) return;
-					const rule = parseFilter(condition.rule);
-					const errors = validatePayload(rule, item, { requireAll: true });
-					return errors.length === 0;
-				});
-
-				if (matchingCondition) {
-					return {
-						...field,
-						meta: merge({}, field.meta || {}, {
-							readonly: matchingCondition.readonly,
-							options: matchingCondition.options,
-							hidden: matchingCondition.hidden,
-							required: matchingCondition.required,
-						}),
-					};
-				}
-
-				return field;
-			} else {
-				return field;
-			}
-		};
-
-		const fieldsWithConditions = fields.value.map((field) => applyConditions(field));
+		const fieldsWithConditions = fields.value.map((field) => applyConditions(rule, field));
```

Comment:
The first argument should be `item` instead of `rule` right? unless I'm misread something here.